test/test-track.sh

```diff
@@ -292,3 +292,136 @@ begin_test "track blocklisted files with glob"
   grep "Pattern .git\* matches forbidden file" track.log
 )
 end_test
+
+begin_test "track lockable"
+(
+  set -e
+
+  repo="track_lockable"
+  mkdir "$repo"
+  cd "$repo"
+  git init
+
+  # track *.jpg once, lockable
+  git lfs track --lockable "*.jpg" | grep "Tracking \*.jpg"
+  numjpg=$(grep -e "\*.jpg.*lockable" .gitattributes | wc -l)
```

Comment:
I've made the `assert_attributes_count` method and it tidies things up, nice idea.